catalog/model/extension/payment/razorpay.php

```diff
@@ -15,4 +15,20 @@ public function getMethod($address, $total)
 
         return $method_data;
     }
+
+    public function setWebhookFlag($order_id,$flag) {
+        
+        $this->db->query("UPDATE " . DB_PREFIX . "order SET webhook_flag = '" . (int)$flag . "' WHERE order_id = '" . (int)$order_id . "'");
```

Comment:
As this table if belongs to opencart, rename the column to `razorpay_webhook_count`